leaf-analysis/src/simulation/BIModelSpecBuilder.java

```diff
@@ -0,0 +1,498 @@
+package simulation;
+
```

Comment:
I think there are a lot of extra lines and odd indents in this file. Not sure if you wanted to change it later/have me point them all out.

Reply:
Yes, this file is only 1/4 done. I'll clean it up for the next PR. :) 
Thanks!!!